### ISSUE TITLE
FCC Badge System

### ISSUE THREAD

**Issue by GJSmith3rd** [FreeCodeCamp/freecodecamp#913]
Please consider adding a badging system to FCC. Mozzilza Open Badges and Pearson Acclaim come to mind.  Badges would be for be granular with rewards for each map assign, section and finally completed FCC coursework and non-profit work.  The younger generation is   
familiar with badges from gaming and this would be a great motivator.  In addition, badging is expected to become more relevant than resumes.

**Comment by LumenTeun** [comment]
This is already being discussed in #604.

**Comment by GJSmith3rd** [comment]
Thanks @LumenTeun for the pointer to #604. I posted a question in Slack @Terakilobye but it was unanswered so I jumped to the waffle and did a quick browse but didn't see any thing similar.  I may have missed it due to lack of  horizontal scrolling and could have used search too.

I'm specifically requesting Open Badges that can be imported/exported, displayed on other sites and loaded into Mozilla BackPack type systems.

Thanks again.
